go/oasis-test-runner/oasis/validator.go

```diff
@@ -146,8 +145,14 @@ func (net *Network) NewValidator(cfg *ValidatorCfg) (*Validator, error) {
 		consensusPort: net.nextNodePort,
 		clientPort:    net.nextNodePort + 1,
 	}
+	net.nextNodePort += 2
 	val.doStartNode = val.startNode
 
+	if cfg.EnableProfiling {
+		val.Node.pprofPort = net.nextNodePort
+		net.nextNodePort++
+	}
```

Comment:
Would be nice if we could unify/cleanup this as it is the same for every `Node`, but this is also true for the `Node` constructor above so probably best for a separate issue/PR.

Reply:
:+1: I'll open an issue